pkg/apis/v1/labels.go

```diff
@@ -99,6 +99,7 @@ var (
 		regexp.MustCompile(fmt.Sprintf("^%s$", regexp.QuoteMeta(NodeClaimTagKey))),
 	}
 	AMIFamilyBottlerocket                          = "Bottlerocket"
+	AMIFamilyBottlerocketFIPS                      = "Bottlerocket-FIPS"
```

Comment:
IIUC there isn't actually a difference in the bootstrapping logic for standard and FIPS bottlerocket AMIs. Given that `spec.amiFamily` doesn't actually dictate AMI selection, I think it's reasonable to not add an additional AMI family for FIPS and just use Bottlerocket. We'd still add the `bottlerocket-fips` alias, it would just map to ami family `Bottlerocket`.